core/src/main/java/org/jdbi/v3/core/statement/internal/JfrSupport.java

```diff
@@ -13,40 +13,44 @@
  */
 package org.jdbi.v3.core.statement.internal;
 
+import java.lang.reflect.Method;
+
 import org.jdbi.v3.core.statement.JdbiStatementEvent;
 
 public final class JfrSupport {
-    private static boolean present = false;
+    private static final boolean JFR_AVAILABLE = ModuleLayer.boot().findModule("jdk.jfr").isPresent();
```

Comment:
This is the way (https://github.com/oracle/graal/commit/cb0f826ef97d8afc1d371bc64066d09620f17f5a#diff-f72b4cc5c4a4201970491910153e0b2ac2ad1b32fb0356dfc732fc331d8f8bc8R54)